terminus-ui/table/src/cell.ts

```diff
@@ -71,20 +85,25 @@ export class TsHeaderCellDefDirective extends CdkHeaderCellDef {}
 })
 export class TsHeaderCellDirective extends CdkHeaderCell {
   constructor(
-    columnDef: CdkColumnDef,
-    elementRef: ElementRef,
+    public columnDef: CdkColumnDef,
     public renderer: Renderer2,
+    private elementRef: ElementRef,
+    @Optional() @Host() @SkipSelf() parent: TsTableComponent<unknown>,
```

Comment:
This means that this child component needs to access parent component's properties?

Reply:
Correct. This is what I had to refactor in the feature I'm working on now. So in the next PR you'll see only parent refer to child.